CSS/CSS.sublime-syntax

```diff
@@ -7,6 +7,14 @@ file_extensions:
   - css.erb
 scope: source.css
 variables:
+  # Many variable names taken directly from https://www.w3.org/TR/CSS22/grammar.html#scanner
+  h: '[0-9a-fA-F]'
+  unicode: '\\{{h}}{1,6}(?:\r\n|[ \t\r\n\f])?'
+  escape: '(?:{{unicode}}|\\[^\r\n\f0-9a-fA-F])'
+  nmstart: '(?:[_a-zA-Z]|[^[:ascii:]]|{{escape}})'
+  nmchar: '(?:[_a-zA-Z0-9-]|[^[:ascii:]]|{{escape}})'
+  ident: '-?{{nmstart}}{{nmchar}}*'
+
```

Comment:
The `mark` & `symbol` categories include characters that need to be escaped — `~!@$%^&*()+=,./';:"?><[]\{}|#​​` etc. — and subcategories & scripts don't offer enough control to match what's needed.

Why not match all non-ASCII except the zero-width & spacing characters like so:

```
nonascii: [^[:ascii:]\x{200B}-\x{200D}\x{FEFF}\p{Z}]
nmstart: ( [_a-zA-Z] | {{nonascii}} | {{escape}} )
nmchar: ( [-\w] | {{nonascii}} | {{escape}} )
```